packages/controller-utils/src/caip.test.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  isEthCaipChainId,
+  buildEthCaipChainId,
+  parseEthCaipChainId,
+  parseEthCaipChainIdHex,
+  parseEthCaipChainIdInt,
+} from './caip';
+
+describe('caip', () => {
+  it('isEthCaipChainId', () => {
+    expect(isEthCaipChainId('eip155:1')).toBe(true);
+    expect(isEthCaipChainId('eip155:5')).toBe(true);
+    expect(isEthCaipChainId('eip155:1337')).toBe(true);
+    expect(isEthCaipChainId('bip122:1')).toBe(false);
+    expect(isEthCaipChainId('eip1555:1')).toBe(false);
+    expect(isEthCaipChainId('eip:1')).toBe(false);
+    expect(isEthCaipChainId('eip:')).toBe(false);
+    expect(isEthCaipChainId('eip155')).toBe(false);
+  });
+
+  describe('buildEthCaipChainId', () => {
```

Comment:
If passed a valid ethCaipChainId will this function just return the arg unmodified?

Reply:
it will throw `Invalid chain ID "${chainId}"`

Reply:
changing this to just return the caip chain id